ingq/book/infra/repository/mysql_book_repository.py

```
@@ -214,3 +214,13 @@ def is_bookmarked(self, user_id: Optional[str], book_id: int) -> Optional[bool]:
             )
 
             return bookmark is not None
+
+    def update_title_image(self, book: Book) -> None:
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)